statics/js/components/graph-layout.js

```diff
@@ -609,6 +611,14 @@ TopologyGraphLayout.prototype = {
       .attr("xlink:href", this.managerImg(d));
   },
 
+  isNeutronNode: function(d) {
```

Comment:
nit- why separate function only for this? not for tun? is this will be used in many places in future?